CMakeLists.txt

```diff
@@ -170,6 +173,16 @@ else()
   target_link_libraries(rccl PUBLIC hip::hip_hcc ${hcc_LIBRARIES} numa)
 endif()
 
+#Setup librccl.so version
+rocm_set_soversion(rccl "2.7")
```

Comment:
SO version for RCCL should be 1.1 I believe.

Reply:
You are right this library version was 1.0, this need to be 1.0